### PR TITLE
Add aria-labeledby in icon

### PR DESCRIPTION
**Pull request by pgurusinga**
Add text on the external standalone link icon 

next steps:
- fetch data from Strapi for custom aria-labelledby text

### CODE REVIEW

File: app/components/OpenInNewTabIcon.tsx [hunk 1]
```diff
@@ -1,8 +1,13 @@
 import OpenInNewIcon from "@digitalservicebund/icons/OpenInNew";
 
+type OpenInNewTabIcon = {
```
Comment (aaschlote) [inline app/components/OpenInNewTabIcon.tsx:3]: can we rename to ``OpenInNewTabIcon`` to ``OpenInNewTabIconProps``? The react component of this file is called also ``OpenInNewTabIcon``?

File: app/components/OpenInNewTabIcon.tsx [hunk 2]
```diff
@@ -15,10 +20,11 @@ export const openInNewAllowedAttributes = {
   path: ["d", "fill"],
 };
 
-export const OpenInNewTabIcon = () =>
+export const OpenInNewTabIcon = ({ ariaLabelledBy }: OpenInNewTabIcon) =>
```
Comment (aaschlote) [inline app/components/OpenInNewTabIcon.tsx:23]: Is it possible to add a small unit test here?

File: app/components/StandaloneLink.tsx [hunk 3]
```diff
@@ -34,7 +34,7 @@ export const StandaloneLink = ({
     <a {...anchorProps}>
       {icon}
       {text}
-      {shouldOpenNewTab && <OpenInNewTabIcon />}
+      {shouldOpenNewTab && <OpenInNewTabIcon ariaLabelledBy={text} />}
```
Comment (aaschlote) [inline app/components/StandaloneLink.tsx:37]: Should be the ``öffnet neues Fenster`` as it's described in the [ticket](https://app.asana.com/0/1207022303033556/1207867713016633)?

**Review by aaschlote — APPROVED** [review]
LGTM

File: app/components/StandaloneLink.tsx [hunk 4]
```diff
@@ -34,7 +34,9 @@ export const StandaloneLink = ({
     <a {...anchorProps}>
       {icon}
       {text}
-      {shouldOpenNewTab && <OpenInNewTabIcon />}
+      {shouldOpenNewTab && (
+        <OpenInNewTabIcon ariaLabelledBy={"In neuem tab öffnen"} />
```
Comment (chohner) [inline app/components/StandaloneLink.tsx:38]: I don't think this is the right usage of `aria-labledby`, as its used to reference an existing element ID in them DOM that contains the label, see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby

Comment (pgurusinga) [reply app/components/StandaloneLink.tsx]: Thanks for the hint! we changed the implementation and tested using the aria-label in the `<a>`, but unfortunately, the voice assistive cannot speak the aria-label inside SVG.

cc: @aaschlote

**Review by chohner — APPROVED** [review]

**Pull request merged**